dlpy/layers.py

```diff
@@ -345,11 +352,11 @@ def _assert_inputs(self, inputs):
                 raise ValueError('Layer {} is called with an input that isn\'t a tensor object'.format(self.name))
 
 
-def Input(n_channels=None, width=None, height=None, name=None, nominals=None, std=None, scale=None,
-          offsets=None, dropout=None, random_flip=None, random_crop=None, random_mutation=None):
+def Input(n_channels=None, height=None, width=None, name=None, nominals=None, std=None, scale=None,
```

Comment:
You need to be careful with changing the order of even keyword parameters.  In Python, you can specify keyword parameters as positional (e.g., Input(0, 1, 2) where 1 was originally width, and now it's height).  Unless you have a really compelling reason, I wouldn't switch the order because it's an API change.

Reply:
The reason I changed the order is that tensor is in either CHW or HWC format. Height is always before width. I assume users tend to specify the height and then the width as they used to if using positional arguments. If we changed so, I think it is a good time to do the change since in previous release there are no predefined models having nonsquare inputs so that it is less likely to break the code. Any suggestion? @sasxianhu @ethem-kinginthenorth